WordPress/Classes/Networking/BlogServiceRemote.h

```diff
@@ -4,9 +4,14 @@
 
 typedef void (^OptionsHandler)(NSDictionary *options);
 typedef void (^PostFormatsHandler)(NSDictionary *postFormats);
+typedef void (^MultiAuthorCheckHandler)(BOOL isMultiAuthor);
 
 @protocol BlogServiceRemote <NSObject>
 
+- (void)checkMultiAuthorForBlog:(Blog *)blog
+                        success:(MultiAuthorCheckHandler)success
+                        failure:(void (^)(NSError *error))failure;
```

Comment:
If the user is not an administrator, this will probably fail both in REST and XML-RPC. Considering that non-admin roles will only happen in multi-author blogs, we should handle this case better :wink: 

To be semantically correct, you're not only interested in 'are there multiple authors', but "can I see other people's posts".

Also, have you considered alternatives to the extra network call? Perhaps only show the everyone/only me selector if there are posts by other people?